### PR TITLE
add .gitignore

### PR DESCRIPTION
**Pull request by ChristofLee**
- Create a gitignore file to hide vscode settings

### CODE REVIEW

**Comment by warengonzaga** [comment]
since you add the .gitignore file can you add more files worth ignoring considering working on WordPress plugins?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.vscode
+node*
+vendor*
```
Comment (warengonzaga) [inline .gitignore:3]: is this any text that contains these texts?

**Review by warengonzaga — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ChristofLee) [reply .gitignore:3]: This would be NPM or Composer packages.

**Review by warengonzaga — APPROVED** [review]
All is good to me, thanks @ChristofLee for your contribution!

**Pull request merged**